app/Services/Phoenix/Phoenix.php

```diff
@@ -31,15 +31,14 @@ private function authenticate()
                 'password' => config('services.phoenix.password'),
             ];
 
-            $response = $this->post($url . 'auth/login', $payload, false);
-
-            $body = json_decode($response->getBody()->getContents(), true);
-            $session_name = $body['session_name'];
-            $session_value = $body['sessid'];
+            $response = $this->post($this->url . 'auth/login', $payload, false);
```

Comment:
So each of these calls can be simplified: `$response = $this->post('auth/login', $payload, false);` 😄